creator-node/sequelize/migrations/20191014204226-add-index-storage-path.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
how much downtime can we expect for content nodes? i know this uses concurrently but does it run in the background to construct the index while it accepts new writes?